docs/user-guide.md

```diff
@@ -2069,6 +2069,14 @@ You can apply Oracle Release Update (RU) or Patch Set Update (PSU) patches to
 both the Grid Infrastructure and Database homes by using the
 `apply-patch.sh` script of the toolkit.
 
+By default, `install-oracle.sh` updates to the latest available patch.  To
+apply a specific patch instead, use `install-oracle.sh`'s `--no-patch` option 
```

Comment:
might be best to avoid the use of the apostrophe on the file name.

"...use the `--no-patch` option in the `install-oracle.sh` file..."